docker/puppetserver-standalone/docker-entrypoint.sh

```diff
@@ -13,4 +26,11 @@ if test -n "${AUTOSIGN}" ; then
   puppet config set autosign "$AUTOSIGN" --section master
 fi
 
+# Allow setting the dns_alt_names for the server's certificate. This
+# setting will only have an effect when the container is started without
+# an existing certificate on the /etc/puppetlabs/puppet volume
+if test -n "${DNS_ALT_NAMES}"; then
+  puppet config set dns_alt_names "${DNS_ALT_NAMES}" --section master
```

Comment:
Should we only run the `puppet config set` if the cert doesn't exist yet? I know practically it has no effect, but could lead to a confusing config file?

Reply:
Good point. The deluxe version would be to also log a warning if DNS_ALT_NAMES is set to something other than what's in `puppet.conf`